cpp/oneapi/dal/algo/knn/infer_types.hpp

```diff
@@ -108,24 +108,39 @@ class infer_result {
         return *this;
     }
 
+    /// Indices of nearest neighbors
+    /// @remark default = table{}
     const table& get_indices() const;
 
     auto& set_indices(const table& value) {
         set_indices_impl(value);
         return *this;
     }
 
+    /// Distances to nearest neighbors
+    /// @remark default = table{}
     const table& get_distances() const;
 
     auto& set_distances(const table& value) {
         set_distances_impl(value);
         return *this;
     }
 
+    /// Set of results that will
```

Comment:
Suggest to remove future tense, consult with @outoftardis

Reply:
won't fix unless @outoftardis propose any particular solution

Reply:
fixed to some extent please check

Reply:
General recommendation from infodev is to stick to present tense, so "Result options that indicates availability of the properties"

Reply:
Applied